app/services/bulk_operation/bulk_update_and_verify_trns.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class BulkOperation::BulkUpdateAndVerifyTrns
```

Comment:
Given the namespace is `BulkOperations` I think its fine to drop the `Bulk` prefix from the class name

Reply:
Or is the prefix to avoid a clash with the data model? If so that rather points back to rich's point that this is an artificial segration of logic

If thats the case, and you want to retain separate services maybe those services should all be moved into their own subnamespace `BulkOperations::Actions` or similar ? Prefixing with `Bulk` feels like a workaround for the namespacing confusion?

Reply:
Though thats probably best left to a follow up PR